CRM/Contribute/Form/Contribution.php

```diff
@@ -2099,7 +2107,8 @@ protected function submit($submittedValues, $action, $pledgePaymentID) {
         $submittedValues['total_amount'] -= $this->_values['tax_amount'] ?? 0;
       }
     }
-    $this->assign('lineItem', !empty($lineItem) && !$isQuickConfig ? $lineItem : FALSE);
+    // This assign is deprecated - it is not used in the templates that ship with core since 2022-23.
+    $this->assign('lineItem', $this->isQuickConfig() ? [$this->getOrder()->getPriceSetID() => $this->getOrder()->getLineItems()] : FALSE);
```

Comment:
@eileenmcnaughton isn't this around the wrong way? i.e. if isQuickConfig it should be set to FALSE right?

Reply:
@seamuslee001 done - hopefully that makes it mergeable